language/stdlib/modules/DesignatedDealer.move

```diff
@@ -84,7 +84,7 @@ module DesignatedDealer {
     ) acquires TierInfo {
         Roles::assert_treasury_compliance(tc_account);
         Roles::assert_designated_dealer(dd);
-        assert(!exists<Dealer>(Signer::address_of(tc_account)), Errors::already_published(EDEALER));
+        assert(!exists<Dealer>(Signer::address_of(dd)), Errors::already_published(EDEALER));
```

Comment:
Good catch! An error I likely introduced.

Reply:
Haha! @tzakian thought he may have been responsible.

Reply:
Please make a note of errors in Move code that you catch with verification. It helps to explain why the Prover project is useful.